packages/mask/src/plugins/Furucombo/constants.ts

```diff
@@ -3,5 +3,5 @@ import { PluginId } from '@masknet/plugin-infra'
 export const PLUGIN_ID = PluginId.Furucombo
 
 export const API_KEY = 'EryftBXrpN4ZG6dViNQtM3uNomPQpeyvgW6El9h0'
-export const API_URL = 'https://api.furucombo.app/v1/investables'
+export const API_URL = 'https://cors.r2d2.to?https://api.furucombo.app/v1/investables'
```

Comment:
What do you think if we do it by configuring the r2d2Fetch method?

Reply:
The background page given the "CORS" error.